src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/ForkedInstance.java

```diff
@@ -39,7 +39,7 @@ public void run()
 
         ProcessUtil.executeScript(config,
                 getStartScriptCommand(),
-                null,
+                ProcessUtil.createEnvironment(null),
```

Comment:
As far as I can tell, this change will not result in any different behavior anywhere in the call stack.